src/test/java/io/cdap/plugin/servicenow/source/ServiceNowSourceConfigTest.java

```diff
@@ -43,8 +56,8 @@
   public void testQueryModeTable() {
     SourceQueryMode queryMode = SourceQueryMode.TABLE;
     ServiceNowSourceConfig config = ServiceNowSourceConfigHelper.newConfigBuilder()
-      .setQueryMode("Table")
-      .build();
+        .setQueryMode("Table")
```

Comment:
why was indent changed? 
you can use this Intellij settings file to format your code - https://cdap.atlassian.net/wiki/spaces/CE/pages/1594098619/Coding+Standard

Reply:
This may have happened while reformatting the file. 
Indentation reverted back. cf1e022323014e4a0be081ac6a3189c04770526a